apps/api/src/view-models/buildViewModelForResource/viewModels/term.view-model.ts

```diff
@@ -28,20 +27,6 @@ export class TermViewModel extends BaseViewModel implements ITermViewModel {
     readonly contributor: string;
 
     // We should wrap the API Property using the View Model Schemas!
-    @ApiProperty({
```

Comment:
Leveraging the `name` property in front-end presenters turned out to be a nice way to gracefully phase-out the old bilingual properties.